solutions/usaco-809.mdx

```diff
@@ -2,14 +2,97 @@
 id: usaco-809
 source: USACO Bronze 2018 Febuary
 title: Taming the Herd
-author: Sathvik Chundru
+author: Sathvik Chundru, Aditya Gupta
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_bronze_feb18.html)
 
 **Time Complexity:** $\mathcal{O}(N)$
 
 <LanguageSection>
+	
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int main()
+{
+	freopen("taming.in", "r", stdin);
+	freopen("taming.out", "w", stdout);
+	int n, min_ans = 0, max_ans = 0;
```

Comment:
assignments on separate lines
also declare variables where you actually use them, `min_ans` isn't used until like way later in the code